go.mod

```diff
@@ -39,8 +39,8 @@ require (
 	github.com/containers/image/v5 v5.17.0 // indirect
 	github.com/containers/libtrust v0.0.0-20190913040956-14b96171aa3b // indirect
 	github.com/containers/ocicrypt v1.1.2 // indirect
-	github.com/containers/psgo v1.7.1 // indirect
-	github.com/containers/storage v1.37.0 // indirect
+	github.com/containers/psgo v1.7.2-0.20220414132323-3ae304491648 // indirect
```

Comment:
You can update to v1.7.2 directly here.

Reply:
Hi @vrothberg 
Thanks for review, updated to v1.7.2.